onedal/covariance/incremental_covariance.py

```diff
@@ -58,10 +58,48 @@ def __init__(self, method="dense", bias=False, assume_centered=False):
         self._reset()
 
     def _reset(self):
+        self._need_to_finalize = False
         self._partial_result = self._get_backend(
             "covariance", None, "partial_compute_result"
         )
 
+    def __getstate__(self):
```

Comment:
I'll add some comments but I guess documentation will be updated in separate PR since currently we don't even have any mention of incremental algos in our docs.

Reply:
We now have this section. I hope this activity already covered by ticket. I am ok to do it in separate PR